core/build.gradle

```diff
@@ -2,10 +2,13 @@ apply from: '../versions.gradle'
 apply plugin: 'com.android.library'
 
 def artifact = "core"
-def versionCodeId = "0.1.2"
+def versionCodeId = 1
+
+project.ext.projectName = "Core"
```

Comment:
What do you think of using a block structure in order to avoid using `project.ext` two times and make it more clear?